src/main/java/stream/flarebot/flarebot/GitHandler.java

```diff
@@ -20,7 +22,15 @@
     static {
         Git git;
         try {
-            git = Git.open(new File("."));
+            git = Git.open(new File("FlareBot/"));
+        } catch (RepositoryNotFoundException e) {
+            try {
+                git =
+                        Git.cloneRepository().setDirectory(new File("FlareBot/")).setURI("https://github.com/FlareBot/FlareBot.git").call();
```

Comment:
I hate this formatting :D

Reply:
Ugh me too, I have yet to find a way to disable it

Reply:
Well mine doesn't do it so IntelliJ formatting settings ;)